packages/react-components/react-tooltip/stories/src/Tooltip/TooltipWithArrowAndFallbackPositions.stories.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import { Checkbox, Tooltip } from '@fluentui/react-components';
+import type { CheckboxProps, PositioningImperativeRef } from '@fluentui/react-components';
+
+export const WithArrowAndFallbackPositions = () => {
```

Comment:
Would be to convert it into a VR story :)

Reply:
Added :)